cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/orion/OrionBackendImpl.java

```diff
@@ -44,177 +53,227 @@
+
+        return response;
     } // deleteSubscriptionV1
-    
+
     @Override
-    public JsonResponse getSubscriptionsByIdV2(String token, String subscriptionId, String fiwareService, 
+    public JsonResponse getSubscriptionsByIdV2(String token, String subscriptionId, String fiwareService,
             String fiwareServicePath) throws Exception {
-        
+
         // create the http header
         ArrayList<Header> headers = getHeaders(token, fiwareService, fiwareServicePath);
-        
+
         String relativeURL = "/v2/subscriptions/" + subscriptionId;
         JsonResponse response = doRequest("GET", relativeURL, true, headers, null);
-        
+
         return response;
     } // getSubscriptionsV2byId
-    
+
     @Override
-    public JsonResponse getSubscriptionsV2(String token, String subscriptionId, String fiwareService, 
+    public JsonResponse getSubscriptionsV2(String token, String subscriptionId, String fiwareService,
             String fiwareServicePath) throws Exception {
-        
-    // create the http header
+
+        // create the http header
         ArrayList<Header> headers = getHeaders(token, fiwareService, fiwareServicePath);
-        
+
         String relativeURL = "/v2/subscriptions";
         JsonResponse response = doRequest("GET", relativeURL, true, headers, null);
-        
+
         return response;
     } // getSubscriptionsV2
-    
+
     @Override
-    public JsonResponse deleteSubscriptionV2(String subscriptionId, String token, String fiwareService, 
+    public JsonResponse deleteSubscriptionV2(String subscriptionId, String token, String fiwareService,
             String fiwareServicePath) throws Exception {
         // create the http header
         ArrayList<Header> headers = getHeaders(token, fiwareService, fiwareServicePath);
-        
+
         String relativeURL = "/v2/subscriptions/" + subscriptionId;
         JsonResponse response = doRequest("DELETE", relativeURL, true, headers, null);
-        
+
         return response;
-        
+
     } // deleteSubscriptionV2
-    
-    private ArrayList<Header> getHeaders (String token, String fiwareService, String fiwareServicePath) {
+
+    private ArrayList<Header> getHeaders(String token, String fiwareService, String fiwareServicePath) {
         ArrayList<Header> headers = new ArrayList<Header>();
         headers.add(new BasicHeader("Content-type", "application/json"));
         headers.add(new BasicHeader("Accept", "application/json"));
         headers.add(new BasicHeader("Fiware-Service", fiwareService));
         headers.add(new BasicHeader("Fiware-ServicePath", fiwareServicePath));
-        
+
         if (token != null) {
             headers.add(new BasicHeader("X-Auth-token", token));
         } // if
-        
+
         return headers;
     }
-    
+
     // TBD: https://github.com/telefonicaid/fiware-cygnus/issues/304
     /**
+     * // @Override public void updateContext(String entityId, String
```

Comment:
Should be this block of commented code (L171 to L207) be removed? Not sure but it could be a leftover...